device-types/HPE/proliant-dl120-g9.yml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: HPE
+model: ProLiant DL120 G9
+slug: hpe-proliant-dl120-g9
+u_height: 1.0
+is_full_depth: true
+airflow: front-to-rear
+part_number: 777426-B21
+comments: '[HPE ProLiant DL120 G9 Server QuickSpecs](https://www.hpe.com/psnow/doc/c04447806)'
+#minumum weight - one hard drive, power supply and processor installed
+weight: 9.0
+weight_unit: kg
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: VGA
+    type: other
+module-bays:
+  - name: PCIe1
+    position: PCIe1
+  - name: PCIe2
+    position: PCIe2
+  - name: PCIe3
+    position: PCIe3
+  - name: PSU1
+    position: PSU1
```

Comment:
It appears that Gen9 has 2 modular PSUs. If that is true, please add the second definition